src/User/LoginForm.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useHistory, Link } from "react-router-dom";
+import Input from "../Input";
+import { loginUser } from "../api";
+import {
+  StyledCardHeading,
+  StyledForm,
+  StyledCard
+} from "../StyledFormComponents";
+
+export const StyledLink = styled(Link)`
+  color: #7a517d;
+  padding: 10px;
+`;
+
+const LoginForm = ({ setCookie }) => {
+  let history = useHistory();
+  const [user, setUser] = useState({
+    username: "",
+    password: ""
+  });
+
+  const handleChange = event => {
+    const { name, value } = event.target;
+    setUser({ ...user, [name]: value });
+  };
+
+  const submitForm = async () => {
+    const newUser = {
```

Comment:
whats the purpose of reassigning this?

Reply:
There doesn't appear to be one so we've just removed that line.